schema/schemas_test.go

```diff
@@ -1,19 +1,27 @@
 package schema
 
 import (
+	"context"
 	"fmt"
 	"io"
 	"net/http"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
+	"time"
 
+	clientutils "github.com/jfrog/jfrog-client-go/utils"
 	"github.com/stretchr/testify/assert"
 	"github.com/xeipuuv/gojsonschema"
 	"gopkg.in/yaml.v3"
 )
 
+const (
+	maxRetries             = 5
+	durationBetweenRetries = 10 * time.Second
```

Comment:
"maxRetries" and "durationBetweenRetries" are out of context.
You can either:
1. Rename it to something like "maxRetriesToDownloadSchema" and "durationBetweenSchemaDownloadRetries"
Or: 
3. Put them in the method